prow/plugins/respond.go

```diff
@@ -26,11 +26,13 @@ import (
 // AboutThisBotWithoutCommands contains the message that explains how to interact with the bot.
 const AboutThisBotWithoutCommands = "Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository."
```

Comment:
FYI I noticed all of the helpers in this file use this one, without the command help. Why? Do we want to standardize on using the full footer?